Scripts/release.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+trap "exit 1" TERM
+export TOP_PID=$$
+
+tag=$1
+
+main() {
+  echo "Release of version $tag is starting"
+  check_tag_existence
+  confirm "Are you sure you want to release this version? (y/n)"
+  confirm "All uncommitted changes will be discarded. Do you want to continue? (y/n)"
+
+  push_tag
+  push_pod
+}
+
+check_tag_existence() {
+  local existing_tag=$(git tag | grep -x "$tag")
+  if [[ $existing_tag != "" ]]
+  then
+    echo "Tag $tag already exists."
+    stop_script
+  fi
+}
+
+push_tag() {
+  git tag $tag
+  local remote=$(get_remote)
+  echo "Pushing tag $tag to $remote"
+  git push $(get_remote) $tag
+  echo "Tag pushed"
+}
+
+push_pod() {
+  confirm "Make sure you are registered using a proper email to CocoaPods trunk. Continue? (y/n)"
+  bundle exec fastlane set_podspec_version version:"$tag"
+  bundle exec pod trunk push ModelValidator.podspec
+  git reset --hard
+  echo "Pod pushed"
+}
+
+confirm() {
+  read -p "$1 " input
+  if [[ $input = "y" || $input = "yes" || $input = "Y" || $input = "YES" ]]
+  then
+    echo "Releasing new version"
```

Comment:
This is not always true. Should we ignore it and simply proceed in this case?

Reply:
We can ignore it